src/components/Callout/MucCallout.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="m-callout" :class="calloutClass" :aria-label="typeAriaLabel">
+    <div class="m-callout__inner">
+      <div class="m-callout__icon">
+        <slot name="icon">
+          <!-- fallback icon -->
+          <component :is="fallbackCalloutIcon" />
+        </slot>
+      </div>
+      <div class="m-callout__body">
+        <div class="m-callout__body__inner">
+          <div class="m-callout__headline">
+            <slot name="header" />
+          </div>
+          <div class="m-callout__content">
+            <slot name="content" class="m-callout__content" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import IconCheck from './icon-check.vue'
+import IconWarning from './icon-warning.vue'
+import IconInformation from './icon-information.vue'
```

Comment:
Can you use the official patternlab-icons according to this doc: https://github.com/it-at-m/muc-patternlab-vue?tab=readme-ov-file#using-mde5-patternlab-icons

"IconWarning" is the only one not available https://patternlab.muenchen.space/?p=viewall-guidelines-icons but could be subsidised by using icon-information again.